examples/darts/uno/uno_example.py

```diff
@@ -0,0 +1,198 @@
+import torch
+import torch.nn as nn
+from torch import optim
+from torch.utils.data import DataLoader
+from loguru import logger
+
+import example_setup as bmk
+import darts
+import candle
+
+
+def initialize_parameters():
+    """ Initialize the parameters for the Uno example """
+
+    uno_example = bmk.UnoExample(
+        bmk.file_path,
+        'default_model.txt',
+        'pytorch',
+        prog='uno_example',
+        desc='Differentiable Architecture Search - Uno example',
+    )
+
+    # Initialize parameters
+    gParameters = candle.finalize_parameters(uno_example)
+    return gParameters
+
+
+def fetch_data(gParameters):
+    """ Download and untar data
+
+    Args:
+        gParameters: parameters from candle
+
+    Returns:
+        path to where the data is located
+    """
+    path = gParameters['data_url']
+    fpath = candle.fetch_file(path + gParameters['train_data'], 'UnoExample')
+    return fpath
+
```

Comment:
`candle.fetch_file` does not seem the be the right way to get the data that Harry shared.